### DISCUSSION TITLE
I can't

### DISCUSSION THREAD

**Discussion by Ominifelix12** [community/community#29994] [Other Feature Feedback, Questions, & Ideas]
How do i use this app

**Comment by radar-blip** [comment]
Can you be more specific? Like, what are you trying to do?

**Reply by Ominifelix12** [reply]
Im trying to change my profile picture.

**Reply by radar-blip** [reply]
Just click on it, then click on 'Edit' and change it.
Or, go into your settings and change it from there. 👍🏼

**Reply by Ominifelix12** [reply]
And how to make use of my sandbox to do my answers after my project works.

**Reply by radar-blip** [reply]
I cant help you with your sandbox, maybe someone else will chime in though.

**Reply by Ominifelix12** [reply]
Ok thank

**Reply by radar-blip** [reply]
You're welcome. Good Luck!

**Reply by dillonstreator** [reply]
@Ominifelix12 Can you be more specific about what you mean by sandbox? Is this related to Github?
